hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotCache.java

```diff
@@ -220,6 +241,8 @@ void cleanup() {
             LOG.debug("Closing SnapshotId {}. It is not being referenced anymore.", k);
             // Close the instance, which also closes its DB handle.
             try {
+              compactSnapshotDB(v.get());
```

Comment:
Please add a wait after this function or inside the funciton and wait for all compactions to finish before you close the db.

Reply:
According to https://github.com/facebook/rocksdb/issues/3335 RocksDatabase.compact() is synchronous so it doesn't need to wait?